recipes/glib/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
+
+package:
+  name: glib
+  version: {{ version }}
+
+source:
+  fn: glib-{{ version }}.tar.xz
+  url: http://ftp.acc.umu.se/pub/GNOME/sources/glib/{{ major_minor }}/glib-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  # patches forked from https://github.com/pkgw/conda-recipes
+  #  Copyright 2014-2015 Peter Williams and collaborators.
+  # This file is licensed under a 3-clause BSD license.
+  patches:
+    - system-data-dirs.patch
+    - no-cocoa-notification.patch  # [osx]
+#    - find-libintl.h.patch  # [osx]
```

Comment:
So yeah, I am glad that we could build without this, but I am not sure we are able to run without it.  

Also, the patch(es) from homebrew are homebrew-specific and will need some updating int order to work, which is part of why I didn't want to fiddle with it.  It will be good if it they get updated though :)